src/Components/common/getUrlInputFile.js

```diff
@@ -0,0 +1,12 @@
+
+export const UrlInput = (e,setUrl,setFieldValue) =>{
+
+  const imageFile = e.target.files[0];
+  const imageUrl = new FileReader();
+  imageUrl.readAsDataURL(imageFile)
```

Comment:
this is one responsability read the file. Maybe it can be a function called readImageFile that returns this line.
```
const readImageFile = (imageFile, onLoadCallback) => {
      const imageUrl = new FileReader();
      imageUrl.readAsDataURL(imageFile);
      imageUrl.onload = (event) => onLoadCallback(event);
}
```

Reply:
I'm sorry, I don't understand what you mean by this, or what do you mean with the responsibility of the form?